source/_code.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  <button aria-controls="<%= name %>-snippets"
```

Comment:
I’m _so happy_ to see accessibility considerations being part of this rebuild.

Something to think about: Is the toggling of code a function of saving screen real estate and therefore the _viewing_ of that code? If so, do we need to the toggle the accessibility of the code to assistive technology (with ARIA here), rather than it just be accessible at all times?

If we were to go that route, then we’d simply need to hide (`aria-hidden`) the toggle button, since it would not need to be used with assistive technology.

This is one of those times where accessibility tools (like ARIA) and usability (with accessibility in mind) create a nuanced situation.

In its current state (hiding and showing the code examples), if we step back and take into consideration the rendering page/DOM as a whole, we have a page outline (the page's headings) of components with a title, but no indication the component's code is available. That is unless you parse all the way through the component and toggle the button.

I don’t think there is a right or wrong answer here, just something to think about.